### DISCUSSION TITLE
Awesome

### DISCUSSION THREAD

**Discussion by simjak** [momegas/megabots#8] [General]
This is more than awesome, I'm new to LangChain and really love the approach you are using. 
Really looking for new features to come, especially tools and agents.

**Comment by momegas** [comment]
Thank you for your nice words @simjak. Feedback and ideas are mega-welcome 😁

**Reply by simjak** [reply]
I will help as much as I can understand :)

**Reply by momegas** [reply]
I'm here to help of course 🤝
Just use the library and report bugs or general feedback

**Reply by simjak** [reply]
```megabots git:(f-readme-fix) ✗  git push --set-upstream origin f-readme-fix
ERROR: Permission to momegas/megabots.git denied to simjak.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.```

I tried to make a small PR

**Reply by momegas** [reply]
Hmm that's strange. What is your remote origin?
